### ISSUE TITLE
Stop the e2e test suite if talkback proxy fails to start

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse#613]
## Problem

The script that runs a proxy server alongside our e2e test suite was added in WordPress/openverse-frontend#881. If for any reason talkback fails to start (for example if there's something already running on port 3000) then the e2e tests still run.

## Description

If talkback fails to start, the whole test run should stop (fail).

> Adding the typical fast-fail mechanisms won't work because we're using & to capture the PID, we'll need specific error handling for that case 😕
(From the [#881 comment](https://github.com/WordPress/openverse-frontend/pull/881#pullrequestreview-889633510))

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.
